Source/Classes/ActionCableClient.swift

```diff
@@ -419,6 +420,8 @@ extension ActionCableClient {
     }
     
     fileprivate func onMessage(_ message: Message) {
+        
+        queue.sync {
```

Comment:
I wonder if this should use `ActionCableSerialQueue` ?

Reply:
@Imperiopolis 

I think sync operation can also be done with `ActionCableSerialQueue`. Btw, thanks for your thoughts.